Server/User/main.py

```
@@ -462,6 +468,74 @@ async def start_weekly_crisis_monitoring():
+        
+        # Check for expiring subscriptions and send reminders
+        reminder_count = subscription_service.check_and_send_expiry_reminders(db)
+        if reminder_count > 0:
+            print(f"💳 Checked subscriptions on startup: {reminder_count} reminder(s) sent")
+        
+        # Check and expire subscriptions
+        expired_count = subscription_service.check_and_update_expired(db)
+        if expired_count > 0:
+            print(f"💳 Checked subscriptions on startup: {expired_count} subscription(s) expired")
+        db.close()
+    except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)